assets/js/components/TourTooltips.js

```diff
@@ -73,27 +74,26 @@ export default function TourTooltips( { steps, tourID } ) {
 	const stepKey = `${ tourID }-step`;
 	const runKey = `${ tourID }-run`;
 	const { setValue } = useDispatch( CORE_UI );
-	const [ shouldSkipTour, setShouldSkipTour ] = useLocalStorage( `sitekit-${ tourID }__has-encountered-tour` );
+	const { dismissTour } = useDispatch( CORE_USER );
```

Comment:
I'm noticing now that #2650 has kind of been forgotten for this release - and it looks like you seem to be implementing that here. Since the PR on #2650 is out of date, maybe we can just keep it in here. cc @eugene-manuilov Can you give this here a pass as well?

Reply:
I didn't even know about that issue and this was a simple change to make that seemed necessary 😄